cmd/web3/main.go

```diff
@@ -553,17 +550,17 @@ func main() {
 									Usage: "Pausable contract.",
 								},
 								cli.BoolFlag{
-									Name:  "mintable, m",
+									Name:  "unmintable, m",
 									Usage: "Mintable contract.",
 								},
 								cli.BoolFlag{
-									Name:  "burnable, b",
+									Name:  "unburnable, b",
```

Comment:
Would it be clearer to leave as `burnable` and default to true?

Reply:
You can't, there's no default value in bool flags, it's always false. I think because they don't require a value on the command line, it's just `--burnable` == `--burnable true`.

Reply:
There is actually  BoolTFlag - https://github.com/gochain-io/web3/blob/master/cmd/web3/main.go#L563

Reply:
Ahh, that's what that is. 😆  I'll try that.

Reply:
That's better, updated to use BoolTFlag.